### PR TITLE
chore: fix some e2e tests

### PR DESCRIPTION
**Pull request by WeizhengSap**
ticket: cxspa-2794

### CODE REVIEW

File: .env-cmdrc [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "CX_BASE_URL": "https://localhost:9002"
   },
   "ci": {
-    "CX_BASE_URL": "https://20.83.184.244:9002"
+    "CX_BASE_URL": "https://spartacus-dev0.eastus.cloudapp.azure.com:9002"
```
Comment (WeizhengSap) [inline .env-cmdrc]: revert it back

**Review by RadhepS — APPROVED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/checkout-as-guest.e2e.cy.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ context('Checkout as guest', { testIsolation: false }, () => {
         .should('eq', 200);
 
       cy.get('cx-login div.cx-login-greet').should('exist');
-      cy.get('.cx-checkout-title').should('contain', 'Delivery Address');
+      cy.get('.cx-checkout-title').should('contain', 'Shipping Address');
 
       cy.get('cx-mini-cart .count').contains('1');
```
Comment (WeizhengSap) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/checkout-as-guest.e2e.cy.ts:59]: Screenshot 2023-03-10 at 3 06 24 PM

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/express-checkout.e2e.cy.ts [hunk 3]
```diff
@@ -41,7 +41,7 @@ context('Express checkout', { testIsolation: false }, () => {
 
       cy.get('cx-mini-cart').click();
       cy.findByText(/proceed to checkout/i).click();
-      cy.get('.cx-checkout-title').should('contain', 'Delivery Address');
+      cy.get('.cx-checkout-title').should('contain', 'Shipping Address');
```
Comment (WeizhengSap) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/express-checkout.e2e.cy.ts:44]: Screenshot 2023-03-10 at 3 08 21 PM

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/site-context/currency/currency-checkout-page.core-e2e.cy.ts [hunk 4]
```diff
@@ -70,7 +70,10 @@ describe('Currency switch - checkout page', () => {
       siteContextSelector.assertSiteContextChange(
         siteContextSelector.FULL_BASE_URL_EN_JPY + checkoutReviewPath
       );
-      cy.get('cx-review-submit .cx-price .cx-value').should('contain', '¥');
+      cy.get('cx-checkout-review-shipping .cx-price .cx-value').should(
+        'contain',
```
Comment (WeizhengSap) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/site-context/currency/currency-checkout-page.core-e2e.cy.ts:74]: Screenshot 2023-03-10 at 3 09 15 PM

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/site-context/language/language-checkout-page.core-e2e.cy.ts [hunk 5]
```diff
@@ -79,7 +79,10 @@ describe('Language switch - checkout page', () => {
         siteContextSelector.FULL_BASE_URL_DE_USD + checkoutReviewPath
       );
 
-      cy.get('cx-review-submit .cx-link').should('contain', deutschName);
```
Comment (WeizhengSap) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/site-context/language/language-checkout-page.core-e2e.cy.ts:82]: Screenshot 2023-03-10 at 3 09 55 PM

File: projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/billing-address.core-e2e-flaky.cy.ts [hunk 6]
```diff
@@ -41,7 +41,7 @@ context('Payment billing address', { testIsolation: false }, () => {
     checkout.fillAddressFormWithCheapProduct();
     checkout.verifyDeliveryMethod();
```
Comment (WeizhengSap) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/billing-address.core-e2e-flaky.cy.ts:42]: Screenshot 2023-03-10 at 3 11 01 PM

**Review by RadhepS — APPROVED** [review]

**Pull request merged**